cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/template/etl/batch/ETLESTest.java

```diff
@@ -101,39 +115,72 @@ public void testESSink() throws Exception {
       .put("format.setting.delimiter", "|")
       .build());
 
-    try {
-      ETLStage sink = new ETLStage("Elasticsearch",
+    ETLStage sink = new ETLStage("Elasticsearch",
+                                 ImmutableMap.of(Properties.Elasticsearch.HOST, "localhost:9200",
```

Comment:
We can check if we can make this programmable. So that if the port is not available, the test doesn't fail.